scipy/stats/rect.pyx

```diff
@@ -1,6 +1,6 @@
 # distutils: language = c++
 
-from Rectangle cimport Rectangle
+from .rect cimport Rectangle
```

Comment:
Depends on the filename; Cython example had Rectangle.pxd but here it is rect.pxd

Reply:
I intended to follow the example but named it incorrectly. Thanks!
But good to know that the `pyx` needs to import from the `pxd`.
I don't understand why the relative import is important, though? I see it doesn't work without it, but I don't understand why.

Reply:
Cython cimport rules are something I don't know a lot about and about which I get burned often.  I can't say in this particular instance why the relative cimport is necessary, but the things I usually try to solve the issue are:
- move the pyx/pxd files to the same directory
- try switching to relative cimport (or removing relative cimport)
- add an `__init__.pxd` file

If these steps don't work, then I'm off to the docs and stackoverflow.  Usually one of the above fixes my issue, but apologies that I'm not sure on this one in particular